contracts/protocols/mainnet/universeFinance/main.sol

```diff
@@ -71,10 +85,37 @@ abstract contract UniverseFinanceResolver is Helpers {
         address universeVault,
         uint256 amount0,
         uint256 amount1
-    ) external view returns(uint256, uint256) {
+    ) external view returns (uint256, uint256) {
         return _depositAmount(universeVault, amount0, amount1);
     }
 
+    /**
+     * @notice get token decimals of a vault
+     * @param vault the vault address
+     * @return token decimals
+     */
+    function decimals(address vault) external view returns (uint8, uint8) {
+        return _decimals(vault);
+    }
+
+    /**
+     * @notice get token decimals of a vault
+     * @param universeVault the vault's address
+     * @param user the user's address
+     */
+    function position(address universeVault, address user)
```

Comment:
Add function to get multiple user Universe Vault position for a user.

Reply:
okay